src/notation/view/pianokeyboard/pianokeyboardcontroller.h

```diff
@@ -47,6 +54,12 @@ class PianoKeyboardController : public muse::async::Asyncable
 
     bool isFromMidi() const;
 
+    bool pianoKeyboardNotatedPitch() const;
+    void setPianoKeyboardNotatedPitch(bool state);
+
+signals:
+    void pianoKeyboardNotatedPitchChanged();
+
```

Comment:
Seems unused; so making this class a `QObject` is also not necessary.

Reply:
We decided to use it but I'm not completely sure if it's necessary or if there's a better way to do it.